src/Components/ExperimentOne/ExperimentOne.js

```diff
@@ -1,12 +1,21 @@
+import ExperimentOneDisplayInformation from "./ExperimentOneDisplayInformation.js";
 import ExperimentOneHeader from "./ExperimentOneHeader.js";
-import ExperimentOneUserForm from "./ExperimentOneUserForm.js";
 
 const ExperimentOne = (props) => {
+  const { isBackButtonClicked } = props;
+  const backButtonStyle = {
```

Comment:
this is a good improvement, but going further and creating a who `Button` reusable component would be better still 😃  let me know if you need help with this

Reply:
I haven't looked into these before but would be curious to learn about this as well. Thakns